contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -13,12 +13,31 @@ contract TrueCreditAgency is UpgradeableClaimable {
 
     uint8 MAX_CREDIT_SCORE = 255;
 
+    struct SavedInterest {
+        uint256 total;
+        uint256 perShare;
+    }
+
+    struct CreditScoreBucket {
+        uint16 borrowersCount;
+        uint128 timestamp;
+        uint256 rate;
+        uint256 cumulativeInterestPerShare; // How much interest was gathered by 1 wei times 10^27
```

Comment:
Not sure what that comment means. If there was accumulated 100 USDC-wei of interest, this variable will store a value of 100*10^27, right?

Reply:
If for each wei you have borrowed, you have to pay 0.1 wei of interest, it will store 0.1e27.

Reply:
👌